### PR TITLE
fixed code smells

### PR DESCRIPTION
**Pull request by larslnde**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EM-5923

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by yogesh-hullatti — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactory.java [hunk 1]
```diff
@@ -210,32 +203,13 @@ private List<CcdValue<CcdBundleDocumentDTO>> addDocumentSet(
...
-        try {
-            return StreamSupport
-                    .stream(list.spliterator(), true)
-                    .map(n -> n.at("/value"))
-                    .filter(n -> anyFilterMatches(documentSelector.filters, n))
-                    .map(node -> this.getDocumentFromNode(node, sortOrder, documentNameValue, documentLinkValue,
-                            customDocumentLinkValue, customDocument))
-                    .collect(Collectors.toList());
-        } catch (Exception ex) {
-            logger.error("addDocumentSet failed,"
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactory.java:222]: can we keep this

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleStitchingService.java [hunk 2]
```diff
@@ -89,9 +83,6 @@ private CcdValue<CcdBundleDTO> stitchBundle(CcdValue<CcdBundleDTO> bundle, CcdCa
             return bundle;
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
-            logger.error(String.format("Stitching Failed for caseId : %s with issue : %s ",
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleStitchingService.java:92]: is it possible to keep this?

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/stitching/mapper/StitchingDTOMapper.java [hunk 3]
```diff
@@ -41,7 +40,9 @@ public StitchingBundleDTO toStitchingDTO(CcdBundleDTO bundleDTO) {
     }
 
     private List<StitchingBundleFolderDTO> getFolders(List<CcdValue<CcdBundleFolderDTO>> folders) {
-        return folders.stream().map(this::getFolder).collect(Collectors.toList());
+        return folders.stream()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/stitching/mapper/StitchingDTOMapper.java:43]: ".stream" can go one to new line like other

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/stitching/mapper/StitchingDTOMapper.java [hunk 4]
```diff
@@ -55,7 +56,9 @@ private StitchingBundleFolderDTO getFolder(CcdValue<CcdBundleFolderDTO> folderDt
     }
 
     private List<StitchingBundleDocumentDTO> getDocuments(List<CcdValue<CcdBundleDocumentDTO>> bundleDocument) {
-        return bundleDocument.stream().map(this::getDocument).collect(Collectors.toList());
+        return bundleDocument.stream()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/stitching/mapper/StitchingDTOMapper.java:59]: ".stream" can go one to new line like other

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/AsyncCcdBundleStitchingServiceTest.java [hunk 5]
```diff
@@ -33,7 +33,7 @@ public class AsyncCcdBundleStitchingServiceTest {
 
     @Before
     public void setup() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/AsyncCcdBundleStitchingServiceTest.java:36]: @initmock annotation does not work ?

**Review by mdayican — APPROVED** [review]

**Pull request merged**